frontend/src/pages/Sessions/SessionsFeedV3/SessionQueryBuilder/components/SessionFeedConfigurationV2/SessionFeedConfigurationV2.tsx

```diff
@@ -46,10 +47,15 @@ export const DropdownMenu = function ({
 		variables: { workspace_id: String(currentWorkspace?.id) },
 		skip: !currentWorkspace?.id,
 	})
+	const { generateSessionsReportCSV, loading: reportLoading } =
+		useGenerateSessionsReportCSV()
 
 	const showDeleteButton =
 		canDelete &&
 		workspaceSettingsData?.workspaceSettings?.enable_data_deletion !== false
+	const showReportButton =
+		workspaceSettingsData?.workspaceSettings?.enable_session_export !==
+		false
```

Comment:
Just want to make sure that if `workspaceSettingsData` was `{}`, then we should want to show the report button?

Reply:
good point - i think originally i thought to make this more lenient in case the workspace settings were cached and had an undefined value, but it still makes sense to be strict in that point.